lib/perl/Genome/Model/ReferenceSequence/Command/Converter/Create.pm

```diff
@@ -5,13 +5,14 @@ use warnings;
 
 use Genome;
 
+use Genome::Model::Build::ReferenceSequence::Converter;
```

Comment:
Now that this uses `valid_values`, it should get loaded by the autoloader without being `use`d here.